amundsen_application/static/js/components/SearchPage/ResourceSelector/index.tsx

```diff
@@ -72,6 +75,7 @@ export class ResourceSelector extends React.Component<ResourceSelectorProps > {
         type: ResourceType.dashboard,
         label: DASHBOARD_RESOURCE_TITLE,
         count: this.props.dashboards.total_results,
+        showBetaFlag: indexDashboardsIsBeta(),
```

Comment:
Same comment about adding in so much temporary code. We could opt to instead check the resourceType in `ResultItemList` to show the `Flag`.